developer/src/kmc-ldml/src/compiler/empty-compiler.ts

```diff
@@ -28,6 +29,45 @@ export class StrsCompiler extends EmptyCompiler {
   public compile(sections: KMXPlus.DependencySections): KMXPlus.Section {
     return new KMXPlus.Strs();
   }
+  public postValidate(section?: KMXPlus.Section): boolean {
+    const strs = <KMXPlus.Strs>section;
+
+    if (strs) {
+      const badStringAnalyzer = new util.BadStringAnalyzer();
+      const CONTAINS_MARKER_REGEX = new RegExp(MarkerParser.ANY_MARKER_MATCH);
```

Comment:
Why ALL_CAPS?

Reply:
as a constant, except it's not. will fix